.github/workflows/build-deps.yml

```diff
@@ -0,0 +1,100 @@
+name: Build dependencies
+
+on:
+  pull_request:
+    branches:
+    - master
+    paths:
+    - .builders/**
+    - datadog_checks_dev/datadog_checks/base/data/agent_requirements.in
+  push:
+    branches:
+    - master
+    paths:
+    - .builders/**
+    - datadog_checks_dev/datadog_checks/base/data/agent_requirements.in
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.head_ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  PYTHON_VERSION: "3.11"
+
+jobs:
+  build:
+    name: Target ${{ matrix.job.image }} on ${{ matrix.job.os }}
+    runs-on: ${{ matrix.job.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+        - os: ubuntu-22.04
+          image: linux-x86_64
+          digest: sha256:cb6bfadf3570f711a5bf1bfe1ce7f33efaab9500b10c7eb469d10c83546399b5
```

Comment:
When and how would we update these digests? Would the idea be to run the workflow once (after a change to the image), get the resulting hash and add a new commit with the updated hash?

Reply:
My bad, I missed the description.